packages/power-assert-context-reducer-ast/index.js

```diff
@@ -28,7 +28,7 @@ module.exports = function (powerAssertContext) {
 function parserOptions(tokens) {
     return {
         sourceType: 'module',
-        ecmaVersion: 7,
+        ecmaVersion: 8,
```

Comment:
Would you use annual version `2017` instead of `8` for [ecmaVersion](https://github.com/ternjs/acorn#main-parser)?